WISE-Community/WISE-Client
# chore(Classroom Monitor): Convert NotebookWorkgroupGrading

## Changes

Converted NotebookWorkgroupGrading from AngularJS to Angular

## Test

1. Go to a run that has the Notebook Notes and Report enabled (or create one)
2. Have students create notes and write a report
3. Go to the Classroom Monitor
4. Go to the Student Notebooks view
5. Make sure the student notes and reports show up and each workgroup can be expanded and collapsed
6. Make sure the "Expand All" and "Collapse All" works

Closes #784
